<?xml version="1.0" encoding="utf-8"?>
<ui version="4.0">
	<class>RegisterWindow</class>
	<widget class="QWidget" name="RegisterWindow">
		<property name="geometry">
			<rect>
				<x>0</x>
				<y>0</y>
				<width>400</width>
				<height>350</height>
			</rect>
		</property>
		<property name="windowTitle">
			<string>Register</string>
		</property>
		<property name="styleSheet">
			<string notr="true">
				QWidget {
				background-color: qlineargradient(x1:0, y1:0, x2:1, y2:1,
				stop:0 #14213d, stop:1 #000000);
				color: white;
				font-family: 'Segoe UI';
				}
				QLabel {
				color: #00d4ff;
				font-size: 16px;
				}
				QLineEdit {
				background-color: #1f4068;
				color: white;
				border: 1px solid #00d4ff;
				border-radius: 8px;
				padding: 5px;
				}
				QPushButton {
				background-color: #1f4068;
				color: white;
				border: 2px solid #00d4ff;
				font-weight: bold;
				border-radius: 12px;
				padding: 6px;
				}
				QPushButton:hover {
				background-color: #ff4d6d;
				border-color: #f72585;
				}
			</string>
		</property>
		<layout class="QVBoxLayout" name="verticalLayout">
			<item>
				<widget class="QLabel" name="label_username">
					<property name="text">
						<string>Username:</string>
					</property>
				</widget>
			</item>
			<item>
				<widget class="QLineEdit" name="usernameEdit"/>
			</item>
			<item>
				<widget class="QLabel" name="label_password">
					<property name="text">
						<string>Password:</string>
					</property>
				</widget>
			</item>
			<item>
				<widget class="QLineEdit" name="passwordEdit">
					<property name="echoMode">
						<enum>QLineEdit::Password</enum>
					</property>
				</widget>
			</item>
			<item>
				<widget class="QLabel" name="label_confirmPassword">
					<property name="text">
						<string>Confirm Password:</string>
					</property>
				</widget>
			</item>
			<item>
				<widget class="QLineEdit" name="confirmPasswordEdit">
					<property name="echoMode">
						<enum>QLineEdit::Password</enum>
					</property>
				</widget>
			</item>
			<item>
				<widget class="QPushButton" name="registerButton">
					<property name="text">
						<string>Register</string>
					</property>
				</widget>
			</item>
		</layout>
	</widget>
	<resources/>
	<connections/>
</ui>
